src/app/table-transactions/table-transactions.component.ts

```diff
@@ -0,0 +1,51 @@
+import { DataSource } from "@angular/cdk/table";
+import { Component, ViewChild } from "@angular/core";
```

Comment:
in all our components we firstly import `Component` and then other data. please, use code-style used in whole app